concent_api/core/subtask_helpers.py

```diff
@@ -283,3 +294,39 @@ def get_one_or_none(
         instances = subtask_or_query_set.filter(**conditions)
         assert len(instances) <= 1
         return None if len(instances) == 0 else instances[0]
+
+
+def finalize_deposit_claim(
+    subtask_id: str,
+    concent_use_case: ConcentUseCase,
+    ethereum_address: str,
+    public_key: str,
+) -> None:
+    deposit_claim = get_one_or_none(
+        DepositClaim,
+        subtask_id=subtask_id,
+        concent_use_case=concent_use_case,
+        payer_deposit_account__ethereum_address=ethereum_address,
+        payer_deposit_account__client__public_key=public_key,
+    )
+
+    if deposit_claim:
```

Comment:
Please compare explicitly, with `is not None`.

Reply:
:+1: done